packages/authorization-process/test/api/setAdminToClient.test.ts

```diff
@@ -0,0 +1,104 @@
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
+import { beforeEach, describe, expect, it, vi } from "vitest";
+import { generateToken, getMockClient } from "pagopa-interop-commons-test";
+import { authRole } from "pagopa-interop-commons";
+import request from "supertest";
+import { generateId, TenantId, UserId } from "pagopa-interop-models";
+import { authorizationApi } from "pagopa-interop-api-clients";
+import { api, authorizationService } from "../vitest.api.setup.js";
+import { clientToApiClient } from "../../src/model/domain/apiConverter.js";
+import {
+  clientAdminAlreadyAssignedToUser,
+  clientKindNotAllowed,
+  clientNotFound,
+  userWithoutSecurityPrivileges,
+} from "../../src/model/domain/errors.js";
+
+describe("API POST /clients/{clientId}/admin test", () => {
+  const mockClient = getMockClient();
+  const body = { adminId: generateId<UserId>() };
+  const makeRequest = async (token: string, clientId: string = mockClient.id) =>
+    request(api)
+      .post(`/clients/${clientId}/admin`)
+      .set("Authorization", `Bearer ${token}`)
+      .set("X-Correlation-Id", generateId())
+      .send(body);
```

Comment:
```suggestion
  const adminSeed = generateId<UserId>();
  const makeRequest = async (
    token: string,
    clientId: ClientId = mockClient.id,
    adminId: UserId = adminSeed
  ) =>
    request(api)
      .post(`/clients/${clientId}/admin`)
      .set("Authorization", `Bearer ${token}`)
      .set("X-Correlation-Id", generateId())
      .send({ adminId });
```
i suggest this change in order to accept `adminId` from input, see my next comment

Reply:
Another way could be [this](https://github.com/pagopa/interop-be-monorepo/pull/1645/files#diff-0c27279763d4c88640c2a0ec01f3fb828731f0673f34892947dcb18e65b7250fR38-R53)